src/react-components/ui-root.js

```diff
@@ -281,12 +280,12 @@ class UIRoot extends Component {
 
       // We are not in mobile chrome, so launch into chrome via an Intent URL
       const location = window.location;
-      const qs = queryString.parse(location.search);
-      qs.vr_entry_type = "daydream"; // Auto-choose 'daydream' after landing in chrome
+      const qs = new URLSearchParams(location.search);
+      qs.set("vr_entry_type", "daydream"); // Auto-choose 'daydream' after landing in chrome
 
       const intentUrl =
         `intent://${location.host}${location.pathname || ""}?` +
```

Comment:
when will `location.pathname` not default to `/`? think you can remove the `|| ""`

Reply:
Good question, reminiscent of the other place [here](https://github.com/mozilla/hubs/pull/466#discussion_r203915251). I will probably remove it.